app/components/Live.vue

```diff
@@ -25,6 +25,9 @@ import { SET_PLAYER_SCREEN, PLAY_URL, PAUSE, SET_PLAYER } from '../store/constan
 export default {
   data: () => {
     return {
+      appBackgroundColor: config.colors.appBackgroundColor,
```

Comment:
We could acquire some code reading quality if we use object destructuring to the config colors attributes.